lib/challenge.rb

```diff
@@ -16,21 +21,12 @@ def each
       next_fib = current_fib + previous_fib
     end
   end
-
-  attr_reader :stop
```

Comment:
It's not a problem that you re-arranged these blocks. But when I write Comparable or Enumerable classes, I personally prefer `<=>` or `each` to be defined before `initialize`.